modules/gdscript/gdscript_compiler.cpp

```diff
@@ -698,11 +698,27 @@ GDScriptCodeGenerator::Address GDScriptCompiler::_parse_expression(CodeGen &code
 											gen->write_call_method_bind(result, base, method, arguments);
 										}
 									} else {
+										// Make temporary if member variable.
```

Comment:
Unsure where to place this, if it's relevant for a greater part of this block, and if it should be more selective (or if we should make a helper function for it), etc.